x.payment_batching/payment_batching.md

```diff
@@ -0,0 +1,307 @@
+of batching and could provide significantly enhanced privacy.  However,
+naive implementations of coinjoin previously provided by Bitcoin
+services have had [flaws][coinjoin sudoku] that prevented them from
+providing significant privacy advantages.  As of February 2019, no
+currently-available coinjoin implementation is fully compatible with the
+needs of payment batching.
+
+## Possible inability to fee bump
+
+A final problem is that you may not be able to fee bump a batched
+payment.  Transaction relay nodes such as Bitcoin Core impose limits on
+the transactions they relay to prevent attackers from wasting bandwidth,
+CPU, and other node resources.  By yourself, you can easily avoid
+reaching these limits, but the receivers of the payments you send can
+respend that money in child transactions that become part of the
+transaction group containing your transaction.
+
+The closer to a limit a transaction group becomes, the less likely
+you'll be able to fee bump your transaction using either
+Child-Pays-for-Parent (CPFP) fee bumping or Replace-by-Fee (RBF) fee
+bumping.  In addition, the more unconfirmed children a transaction has,
+the more RBF fee bumping will cost as you'll have to pay for both the
+increased feerate of your transaction as well as for all the potential
+fees lost to miners when they remove any child transactions in order
+to accept your replacement.
+
+Note that these problems are not unique to batched payments---independent
+payments can have the same problem.  However, if an independent payment
+can't be fee bumped because the independent receiver spent their output,
+only that user is affected.  But if a single receiver of a batched
+payment spends their output to the point where fee bumping becomes
+impossible, all the other receivers of that transaction are also affected.
+
+## Implementation
+
+Payment batching is extremely easy using certain existing wallet
+implementations, such as Bitcoin Core.  Check your software
+documentation for a function that allows you to send multiple payments.
+
+```bash
+bitcoin-cli sendmany "" '{
+  "bc1q5c2d2ue7x38hcw2ugk5q7y4ae7nw4r6vxcptu8": 0.1,
+  "bc1qztjzd7hpf2xmngr7zkgkxsvdqcv2jpyfgwgtsv": 0.2,
+  "bc1qsul9emtnz0kks939egx2ssa6xnjpsvgwq9chrw": 0.3
+}'
+```
+
+<!-- for max standard tx size: src/policy/policy.h:static const unsigned int MAX_STANDARD_TX_WEIGHT = 400000; -->
+
+If using your own implementation, you are probably already creating
+transactions with two outputs in most cases, so it should be easy to add
+support for additional outputs.  The only notable consideration is that
+Bitcoin Core nodes (and most other nodes) will refuse to accept or relay
+transactions over 100,000 vbytes, so you should not attempt to send
+batched payments larger than this.
+
+## Combining batching with other opt-in scaling techniques
+
+Payment batching works well with all the techniques described in this
+guide, but some combinations are especially notable:
```

Comment:
I like the sounds of "this guide" but in order not to introduce different ways of saying the same thing, should we instead say "this book" or "in this scaling book"?

Reply:
Agree. I like the sound of guide, but think we should standardize to book for now.